HACKING.md

```diff
@@ -212,6 +212,20 @@ oc annotate secret/azure-kola-tests-config \
     jenkins.io/credentials-description="Azure kola tests credentials config"
 ```
 
+#### [OPTIONAL] Using shared Azure managed identity for kola tests
+If you want to establish an authorized identity to pre-emptively grant access
+to other Azure resources, enabling the startup process to leverage Azure's default
+credentials, use the azure-kola-managed-identity:
+
+```
+oc create secret generic azure-kola-managed-identity \
+    --from-file=text=/run/user/$UID/azureManagedIdentity.txt
```

Comment:
I should have looked more closely at other secrets when we were defining this one earlier. It looks like we could define it a little easier:
```
MANAGED_IDENTITY_ID=<azure managed identity resource id>
oc create secret generic azure-kola-managed-identity  --from-literal=text="${MANAGED_IDENTITY_ID}"
```

This also defines the format of the secret, which is consistent the rest of this doc.